Source/com/drew/metadata/mp4/boxes/UserDataBox.java

```diff
@@ -44,6 +44,7 @@
     public UserDataBox(@NotNull final SequentialReader reader, @NotNull final Box box, int length) throws IOException {
         super(box);
 
+        // TODO: This logic should be added to ItemListBox by adding a new item for "©xyz"
```

Comment:
This is the original location support that Drew added a while back. Since we are implementing the ItemListbox, it can actually be moved to the new ItemListBox file.